src/utils/format-date.js

```diff
@@ -35,7 +35,7 @@ export function formatDate(d) {
     formattedLocal: `${months[date.getMonth()]} ${date.getDate()}, ${date.getFullYear()} ${localTimeTwelve.join(':')} ${localAMPM} (UTC ${localOffset})`, // local time
     formattedLocalShort: `${months[date.getMonth()]} ${date.getDate()}, ${date.getFullYear()} (UTC ${localOffset})`, // local time
     full: date.toUTCString(),
-    timestamp: date.getTime() / 1000
+    timestamp: date.getTime()
```

Comment:
`date.getTime()` returns millis, so it looks like this was what you wanted prior to this change as it will now set `timestamp` to the full milliseconds from epoc.

Reply:
My best guess as to what is happening here is the date passed into this helper method already has the millis trimmed.

Reply:
Yes, I agree

There are so many / 1000 in the UI that I got a bit confused.